### ISSUE TITLE
isMeters variable name(s)

### ISSUE THREAD

**Issue by RaananW** [immersive-web/image-tracking#4]
There are two parameters in the proposed IDL that have `inMeters` in their name: `measuredWidthInMeters` and `widthInMeters`.

As XR units are already in meters, and other APIs don't have this name definitions, is that needed in this API?

**Comment by klausw** [comment]
I'm open to suggestions, but I think that an attribute named `width` that's provided alongside a bitmap image is too easy to misinterpret as a pixel width. It's not immediately clear from context that this is intended to be a real-world dimension.

**Comment by RaananW** [comment]
Yep, you are right. Apart from `realWorldWidth` or `worldWidth`  that corresponds with `worldInformation` from plane detection (which is also not ideal) I don't have any better suggestion.
